### ISSUE TITLE
Falky spec : `accessibility/agents_pages_spec.rb`

### ISSUE THREAD

**Issue by damienlethiec** [betagouv/rdv-solidarites.fr#2952]
La spec `spec/features/accessibility/agents_pages_spec.rb:19` est flaky.

Il semblerait que l'apparition de la flakiness soit corrélée avec #2894.

```
rspec ./spec/features/accessibility/agents_pages_spec.rb:19

  1) agents page agenda with 3 rdvs is accessible
     Failure/Error: expect(page).to be_axe_clean

       Found 1 accessibility violation:

       1) color-contrast: Elements must have sufficient color contrast (serious)
           https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=axeAPI
           The following 2 nodes violate this rule:
           
               Selector: .fc-time-grid-event.fc-event-unknown[title=""]:nth-child(1) > .fc-content > .fc-time[data-start="\31 2\:01"][data-full="\31 2\:01\ -\ 12\:46"] > span
               HTML: <span>12:01 - 12:46</span>
               Fix any of the following:
               - Element has insufficient color contrast of 2.35 (foreground color: #ffffff, background color: #a9a9a9, font size: 8.3pt (11.05px), font weight: normal). Expected contrast ratio of 4.5:1
               
               Selector: .fc-time-grid-event-inset.fc-time-grid-event.fc-event-unknown:nth-child(2) > .fc-content > .fc-time[data-start="\31 2\:01"][data-full="\31 2\:01\ -\ 12\:46"] > span
               HTML: <span>12:01 - 12:46</span>
               Fix any of the following:
               - Element has insufficient color contrast of 2.35 (foreground color: #ffffff, background color: #a9a9a9, font size: 8.3pt (11.05px), font weight: normal). Expected contrast ratio of 4.5:1
```
